### DISCUSSION TITLE
Duda formulario

### DISCUSSION THREAD

**Discussion by Lumpalups** [LaunchX-InnovaccionVirtual/CursoIntroPython#241] [Q&A] [answered]
¿Alguien tiene problemas para ingresar al formulario? A mi me aparece lo siguiente![image](https://user-images.githubusercontent.com/92233811/153935950-54970426-2e58-46a0-b18e-f9cd7b8086d3.jpeg) y ya me cercioré de que sea la cuenta de launch x que nos proporcionaron :/

**Comment by FernandaOchoa** [comment, marked as answer]
Hola, debes ingresar desde una pestaña de incógnito o pestaña privada en tu navegador.

**Reply by Lumpalups** [reply]
muchas gracias🥺🖤
